### PR TITLE
Fixed linebreak

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: doc/symfony-cache-configuration.rst [hunk 1]
```diff
@@ -168,7 +168,8 @@ options through the constructor:
 
     With Apache, you can do this for example in a ``.htaccess`` file:
```
Comment (ddeboer) [inline doc/symfony-cache-configuration.rst]: I guess this should end in `::` instead to make the snippet below a proper code sample.

**Comment by ddeboer** [comment]
Thanks! While you’re at it, could you also change the colon line ending to make this into a proper code snippet?

**Comment by Nyholm** [comment]
Of course. I was not familiar with that syntax. 
I've updated the file.

**Comment by dbu** [comment]
thanks @Nyholm - are you giving the FOSHttpCache a try or already using it?

**Comment by Nyholm** [comment]
Thank you for merging. 
I'm giving it a try. I was stuck for some time due a [bug in SensioFrameworkExtraBundle](https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/349) but apart from that the integration was easy.

**Comment by dbu** [comment]
cool, glad you like it. the framework extra bundle thing sounds like a
problem indeed. might even be a security issue when using the user context.

**Pull request merged**